### PR TITLE
Added Drupal 10 support

### LINKED ISSUE #42: Drupal 10 support
Issue to track any changes needed to support Drupal 10

### PR DESCRIPTION
**Pull request by stephen-cox**
Fixes #42

### CODE REVIEW

**Comment by finnlewis** [comment]
One cause of the failing tests appears to be that jQuery once() is gone from Drupal 10. 

https://www.drupal.org/forum/support/upgrading-drupal/2023-06-21/jquery-once-in-drupal-10

I made a start at trying to fix that, but I'm no JS expert and I'm not sure if the address lookup was working for me in the first place! 

@Adnan-cds should the address lookup work out of the box for me on a D9 install?

**Comment by Adnan-cds** [comment]
> should the address lookup work out of the box for me on a D9 install?

Yeah, it should work.  Otherwise the Drupal 9 test wouldn't have passed.

> One cause of the failing tests appears to be that jQuery once() is gone from Drupal 10.

I am working on this on a different module.  I can take a look at this later if you want.

File: js/address_change.js [hunk 1]
```diff
@@ -80,7 +80,7 @@
     attach: function (context, settings) {
 
       // Behaviors to only attach once.
-      $('.js-centralhub-address-lookup', context).once('central-hub-behaviours').each(function () {
+      $(once('central-hub-behaviours', '.js-centralhub-address-lookup', context)).each(function () {
```
Comment (Adnan-cds) [inline js/address_change.js:83]: Interesting :)

**Review by Adnan-cds — APPROVED** [review]
Tests are passing, so why not :)

**Pull request merged**